data/missilewars/functions/game_modes/4teams/load_map.mcfunction

```diff
@@ -14,17 +14,26 @@ execute at @e[type=marker,tag=Base,x=0] if score Map m.Variables matches 1 run f
 
 execute at @e[type=marker,tag=Base,x=0] if score Map m.Variables matches 1 positioned ^-25 ^ ^4 run fill ~ 0 ~ ^50 ^-1 ^4 minecraft:white_stained_glass
 
-execute at @e[type=marker,tag=BlueBase,x=0] if score Map m.Variables matches 2..5 run function missilewars:maps/blue/basic_map
-execute at @e[type=marker,tag=GreenBase,x=0] if score Map m.Variables matches 2..5 run function missilewars:maps/green/basic_map
-execute at @e[type=marker,tag=RedBase,x=0] if score Map m.Variables matches 2..5 run function missilewars:maps/red/basic_map
-execute at @e[type=marker,tag=PurpleBase,x=0] if score Map m.Variables matches 2..5 run function missilewars:maps/purple/basic_map
+execute at @e[type=marker,tag=BlueBase,x=0] if score Map m.Variables matches 2..6 run function missilewars:maps/blue/basic_map
+execute at @e[type=marker,tag=GreenBase,x=0] if score Map m.Variables matches 2..6 run function missilewars:maps/green/basic_map
+execute at @e[type=marker,tag=RedBase,x=0] if score Map m.Variables matches 2..6 run function missilewars:maps/red/basic_map
+execute at @e[type=marker,tag=PurpleBase,x=0] if score Map m.Variables matches 2..6 run function missilewars:maps/purple/basic_map
 execute if score Map m.Variables matches 2 run fill -25 0 -2 25 100 2 white_stained_glass
 execute if score Map m.Variables matches 2 run fill -2 0 -25 2 100 25 white_stained_glass
 
 execute at @e[type=marker,tag=Base,x=0] if score Map m.Variables matches 3 run fill ^-25 ^ ^23 ^25 ^ ^27 white_stained_glass
 execute at @e[type=marker,tag=Base,x=0] if score Map m.Variables matches 3 run fill ^-25 ^-20 ^23 ^25 ^-20 ^30 white_stained_glass
 execute at @e[type=marker,tag=Base,x=0] if score Map m.Variables matches 3 run fill ^-25 ^-40 ^23 ^25 ^-40 ^32 white_stained_glass
 
+execute at @e[type=marker,tag=Base,x=0] if score Map m.Variables matches 6 run fill ^11 ^8 ^12 ^17 ^8 ^18 white_stained_glass
+execute at @e[type=marker,tag=Base,x=0] if score Map m.Variables matches 6 run fill ^15 ^1 ^14 ^15 ^8 ^16 white_stained_glass
+execute at @e[type=marker,tag=Base,x=0] if score Map m.Variables matches 6 run fill ^13 ^1 ^14 ^13 ^8 ^16 white_stained_glass
+execute at @e[type=marker,tag=Base,x=0] if score Map m.Variables matches 6 run fill ^14 ^1 ^15 ^14 ^8 ^15 ladder[facing= west ]
+execute at @e[type=marker,tag=Base,x=0] if score Map m.Variables matches 6 run fill ^-11 ^8 ^12 ^-17 ^8 ^18 white_stained_glass
+execute at @e[type=marker,tag=Base,x=0] if score Map m.Variables matches 6 run fill ^-15 ^1 ^14 ^-15 ^8 ^16 white_stained_glass
+execute at @e[type=marker,tag=Base,x=0] if score Map m.Variables matches 6 run fill ^-13 ^1 ^14 ^-13 ^8 ^16 white_stained_glass
+execute at @e[type=marker,tag=Base,x=0] if score Map m.Variables matches 6 run fill ^-14 ^1 ^15 ^-14 ^8 ^15 ladder[facing= west ]
+
```

Comment:
Same comments as the other section. If you combine these commands into a function you can avoid repeating this code.